### DISCUSSION TITLE
How to get the picture for the spleen_segmentation_3d?

### DISCUSSION THREAD

**Discussion by timeanddoctor** [Project-MONAI/MONAI#3476] [Q&A]
How to get the picture for the spleen_segmentation_3d segement?

![image](https://user-images.githubusercontent.com/28825816/145672767-31b6f303-30a1-4b39-b8d6-c5fc91e1995c.png)

**Comment by Nic-Ma** [comment]
Hi @timeanddoctor ,

Sorry I didn't get what you mean, could you please help share more details about your question?
Thanks in advance.

**Reply by vikashg** [reply]
Probably @timeanddoctor means an isometric view of the segmented spleen. Or a surface mesh that can be viewed in 3D ?

**Comment by timeanddoctor** [comment]
Thanks ,@Nic-Ma @vikashg  

What I am doing is to get a U-net algorithm figure to explain the Unet progress of the spleen_segmentation.  I am not sure I can draw the U net figure from this code parameters :
```

self._model = UNet(
            spatial_dims=3,
            in_channels=1,
            out_channels=2,
            channels=(16, 32, 64, 128, 256),
            strides=(2, 2, 2, 2),
            num_res_units=2,
            norm=Norm.BATCH,
        )
```

Thanks for your help.

**Comment by timeanddoctor** [comment]
Is there any library that can visualize the whole process of MONAI.

**Reply by Nic-Ma** [reply]
Hi @timeanddoctor ,

I would suggest you to take a look at this project: https://github.com/TylerYep/torchinfo.

Thanks.
